src/read/read_ref.rs

```diff
@@ -39,6 +39,22 @@ where
         }
     }
 
+    /// Iterate over all keys in the map.
```

Comment:
One thing I found interesting ... which is just off screen, is that `iter` is `Some(self.guard.data.iter())` ... why is it optional? Is it ever `None`?

Reply:
Hmm, that might be leftover from before we had `read()` and `MapReadRef`. If the `Option` can be removed now, then please do!

Reply:
c79d3c027e68c49346bcd2178cbd7708836383b2